src/main/java/com/google/api/codegen/transformer/PackageMetadataTransformer.java

```diff
@@ -39,6 +39,7 @@
         .protoVersionBound(packageConfig.protoVersionBound(language))
         .commonProtosVersionBound(packageConfig.commonProtosVersionBound(language))
         .authVersionBound(packageConfig.authVersionBound(language))
+        .packageName(packageConfig.packageName(language))
```

Comment:
Hmm, I don't see that change; did you push the commit? (Also, is the `packageName` field of `PackageMetadataView` used anywhere now that `protoPackageName` is available?)